data/blobs/blobs.json

```diff
@@ -0,0 +1,730 @@
+{
```

Comment:
This is as a json and in SQL, is there a reason for this? Or is one of these vestigial?

Reply:
Deleted the JSON file, it was leftover from previous way of getting blobs.